development/build/constants.js

```diff
@@ -0,0 +1,48 @@
+const TASKS = {
```

Comment:
Good idea to put all of these in one place, helps a lot to understand the big picture.